lib/services/local/generic/generic.go

```diff
@@ -121,6 +121,19 @@ func (s *Service[T]) WithPrefix(parts ...string) *Service[T] {
 	}
 }
 
+// CountResources will return a count of all resources in the prefix range.
+func (s *Service[T]) CountResources(ctx context.Context) (int, error) {
+	rangeStart := backend.ExactKey(s.backendPrefix)
+	rangeEnd := backend.RangeEnd(rangeStart)
+
+	result, err := s.backend.GetRange(ctx, rangeStart, rangeEnd, backend.NoLimit)
```

Comment:
I understand the reason but this might create OOM situations where we load a substantial number of objects and its JSON payload into memory. For members it might be fine but if someone else tries to run this method with larger objects it will be serious problem.

Reply:
I'll paginate here based on the page limit set by the generic service.